doc/manual/source/interaction/web-api.rst

```diff
@@ -117,46 +117,46 @@ Porcelain commands supported
 
 - **Checkout** (-T)
 
-	 **Currently Supported Options:**
+    **Currently Supported Options:**
 
-		a) **branch*** - the name of the branch to checkout
+      a) **branch** - the name of the branch to checkout
 
-			**Type:** String
+      	**Type:** String
```

Comment:
The `Type` and `Default` sections here aren't indented enough.
Current:
![current1](https://cloud.githubusercontent.com/assets/1253074/25403695/cf1f06ae-29c2-11e7-8b7a-dbcba790b474.png)
Edit:
![edit1](https://cloud.githubusercontent.com/assets/1253074/25403696/cf249934-29c2-11e7-9142-c833cf82feba.png)